Visitor.NET.AutoVisitableGen/Visitor.NET.AutoVisitableGen/AutoVisitableIncrementalSourceGenerator.cs

```diff
@@ -81,6 +84,17 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
             return null;
 
         var visitableName = typeDeclarationSyntax.Identifier.Text;
+        
+        
+        // check all types in hierarchy are partial
+        SyntaxNode currentType = typeDeclarationSyntax;
+        while (currentType is not BaseNamespaceDeclarationSyntax)
+        {
+            if (currentType is BaseTypeDeclarationSyntax b && !b.Modifiers.Any(SyntaxKind.PartialKeyword))
+                return null;
+
+            currentType = currentType.Parent;
+        }
```

Comment:
I don't think it is a good idea to require the whole hierarchy to consist of partial classes.

The point of library is the flexibilty and the freedom of choice - source gen usage only for the selected types

Reply:
No, you are right, maybe using the word "hierarchy" was confusing.  I ment to the outer type(s).

But now I think we can remove this check, let the compiler show an error.
When removing the `partial` keyword from `Operation` in the Sample, there is the following error :
`Duplicate definition 'Visitor.NET.AutoVisitableGen.Sample.Operation'. Possibly missing keyword 'partial'`.
What do you think ?

Reply:
Agree, remove the check

Reply:
removed